syntax = "proto3";

enum ProtoStorageError {
  NO_ERRORS = 0;
  INVALID_TASK = 1;
  TASK_NOT_FOUND = 2;
  PARENT_NOT_FOUND = 3;
  WRONG_TASK_ID = 4;
}